pyramid_oereb/routes.py

```diff
@@ -12,18 +12,18 @@ def includeme(config):  # pragma: no cover
     """
 
     # Service for logo images
-    config.add_route('{0}/image/logo'.format(route_prefix), '/image/logo/{logo}')
+    config.add_route('{0}/image/logo'.format(route_prefix), '/image/logo/{logo}.png')
     config.add_view(Logo, attr='get_image', route_name='{0}/image/logo'.format(route_prefix),
                     request_method='GET')
 
     # Service for municipality images
-    config.add_route('{0}/image/municipality'.format(route_prefix), '/image/municipality/{fosnr}')
+    config.add_route('{0}/image/municipality'.format(route_prefix), '/image/municipality/{fosnr}.png')
     config.add_view(Municipality, attr='get_image', route_name='{0}/image/municipality'.format(route_prefix),
                     request_method='GET')
 
     # Service for symbol images
     config.add_route('{0}/image/symbol'.format(route_prefix),
-                     '/image/symbol/{theme_code}')
+                     '/image/symbol/{theme_code}/{type_code}.png')
```

Comment:
I'm not sure if we should hardcode the image type or leave it configurable...

Reply:
OK. According to the specification, symbol are allowed as PNG and SVG. I will remove the format extension.